docs/built_rst/csv/elements/polling_location.rst

```diff
@@ -11,11 +11,11 @@ The PollingLocation object represents a site where voters cast or drop off ballo
 | Tag             | Data Type                | Required?    | Repeats?     | Description                              | Error Handling                           |
 +=================+==========================+==============+==============+==========================================+==========================================+
 | address_line    | ``xs:string``            | **Required** | Repeats      | Represents the various parts of an       | At least one valid ``AddressLine`` must  |
-|                 |                          |              |              | address to a polling location. For CSV   | be present for ``PollingLocation`` to be |
-|                 |                          |              |              | implementations, repeating               | valid. If no valid ``AddressLine`` is    |
-|                 |                          |              |              | ``address_line`` columns should be       | present, the implementation is required  |
-|                 |                          |              |              | incremented, e.g. address_line1,         | to ignore the ``PollingLocation``        |
-|                 |                          |              |              | address_line2, address_line3.            | element containing it.                   |
+|                 |                          |              |              | address to a polling location.           | be present for ``PollingLocation`` to be |
```

Comment:
Why were the address_line documentation notes removed?

Reply:
The notes were incorrect. CSV implementation does support address_line1, address_line2 fields. Only one address_line is allowed.